ui/eslint.config.js

```diff
@@ -8,12 +8,7 @@ const config = defineConfig([
   ...createConfig(opts),
   {
     rules: {
-      'jsx-a11y/anchor-is-valid': 'off',
-      'jsx-a11y/control-has-associated-label': 'off',
-      'react/destructuring-assignment': 'off',
-      'react/no-multi-comp': 'off',
-      'react/no-unused-class-component-methods': 'off',
```

Comment:
I've kept all these dead methods for now in case they became dead by mistake (i.e. regressions). Again, something to look at when refactoring class components.